server/src/main/java/dev/slimevr/protocol/rpc/RPCHandler.java

```diff
@@ -257,6 +259,17 @@ public void onClearDriftCompensationRequest(
 		this.api.server.clearTrackersDriftCompensation();
 	}
 
+	public void onGUIInfosResponse(
+		GenericConnection conn,
+		RpcMessageHeader messageHeader
+	) {
+		GUIInfosResponse req = (GUIInfosResponse) messageHeader.message(new GUIInfosResponse());
+		if (req == null)
+			return;
+
+		this.api.server.setLegTweaksEnabled(!req.inProportions());
```

Comment:
What happens if you have legtweaks disabled in the config? Does this enable it even if you disabled it manually?

Reply:
nope! setLegTweaksEnabled is the master on/off for all 4 options (skating correction, floor clip, foot plant, and toe snap), It is the same way Autobone turns them off

Reply:
All good, just wanted to double check